Tests are failing with make

When I do the normal steps of

1. ./autogen.sh
2. ./configure
3. make

The tests seem to be failing because of linking.

log:

```
  In file included from src/test/../../include/utreexo.h:4,
                   from src/test/accumulator_tests.cpp:1:
  src/test/../../include/accumulator.h:9:25: error: ‘uint8_t’ was not declared in this scope
      9 | using Hash = std::array<uint8_t, 32>;
        |                         ^~~~~~~
  src/test/../../include/accumulator.h:9:36: error: template argument 1 is invalid
      9 | using Hash = std::array<uint8_t, 32>;
        |                                    ^
  src/test/../../include/accumulator.h:10:24: error: ‘Hash’ was not declared in this scope
     10 | using Leaf = std::pair<Hash, bool>;
        |                        ^~~~
  src/test/../../include/accumulator.h:10:34: error: template argument 1 is invalid
     10 | using Leaf = std::pair<Hash, bool>;
        |                                  ^
  src/test/../../include/accumulator.h:18:25: error: expected ‘)’ before ‘num_leaves’
     18 |     Accumulator(uint64_t num_leaves);
        |                ~        ^~~~~~~~~~~
        |                         )
  src/test/../../include/accumulator.h:25:61: error: ‘Hash’ was not declared in this scope
     25 |     virtual bool Prove(BatchProof& proof, const std::vector<Hash>& target_hashes) const = 0;
        |                                                             ^~~~
  src/test/../../include/accumulator.h:25:65: error: template argument 1 is invalid
     25 |     virtual bool Prove(BatchProof& proof, const std::vector<Hash>& target_hashes) const = 0;
        |                                                                 ^
  src/test/../../include/accumulator.h:25:65: error: template argument 2 is invalid
  src/test/../../include/accumulator.h:35:35: error: ‘Leaf’ was not declared in this scope
     35 |     bool Modify(const std::vector<Leaf>& new_leaves, const std::vector<uint64_t>& targets);
        |                                   ^~~~
  src/test/../../include/accumulator.h:35:39: error: template argument 1 is invalid
     35 |     bool Modify(const std::vector<Leaf>& new_leaves, const std::vector<uint64_t>& targets);
        |                                       ^
  src/test/../../include/accumulator.h:35:39: error: template argument 2 is invalid
  src/test/../../include/accumulator.h:35:72: error: ‘uint64_t’ was not declared in this scope
     35 |     bool Modify(const std::vector<Leaf>& new_leaves, const std::vector<uint64_t>& targets);
        |                                                                        ^~~~~~~~
  src/test/../../include/accumulator.h:35:80: error: template argument 1 is invalid
     35 |     bool Modify(const std::vector<Leaf>& new_leaves, const std::vector<uint64_t>& targets);
        |                                                                                ^
  src/test/../../include/accumulator.h:35:80: error: template argument 2 is invalid
  src/test/../../include/accumulator.h:38:28: error: ‘Hash’ was not declared in this scope
     38 |     void Roots(std::vector<Hash>& roots) const;
        |                            ^~~~
  src/test/../../include/accumulator.h:38:32: error: template argument 1 is invalid
     38 |     void Roots(std::vector<Hash>& roots) const;
        |                                ^
  src/test/../../include/accumulator.h:38:32: error: template argument 2 is invalid
  src/test/../../include/accumulator.h:57:5: error: ‘uint64_t’ does not name a type
     57 |     uint64_t m_num_leaves;
        |     ^~~~~~~~
  src/test/../../include/accumulator.h:66:53: error: ‘uint64_t’ has not been declared
     66 |     virtual NodePtr<Accumulator::Node> SwapSubTrees(uint64_t from, uint64_t to) = 0;
        |                                                     ^~~~~~~~
  src/test/../../include/accumulator.h:66:68: error: ‘uint64_t’ has not been declared
     66 |     virtual NodePtr<Accumulator::Node> SwapSubTrees(uint64_t from, uint64_t to) = 0;
        |                                                                    ^~~~~~~~
  src/test/../../include/accumulator.h:73:50: error: ‘uint64_t’ has not been declared
     73 |     virtual NodePtr<Accumulator::Node> MergeRoot(uint64_t parent_pos, Hash parent_hash) = 0;
        |                                                  ^~~~~~~~
  src/test/../../include/accumulator.h:73:71: error: ‘Hash’ has not been declared
     73 |     virtual NodePtr<Accumulator::Node> MergeRoot(uint64_t parent_pos, Hash parent_hash) = 0;
        |                                                                       ^~~~
  src/test/../../include/accumulator.h:75:54: error: ‘Leaf’ does not name a type
     75 |     virtual NodePtr<Accumulator::Node> NewLeaf(const Leaf& leaf) = 0;
...
```

--- a/include/accumulator.h
+++ b/include/accumulator.h
@@ -2,6 +2,7 @@
 #define UTREEXO_ACCUMULATOR_H
 
 #include <array>
+#include <stdint.h>
 #include <utility>
 #include <vector>
 

